docs/CHANGELOG.md

```diff
@@ -12,10 +12,23 @@ For more information about `AlexsLemonade/scpca-nf` versions, please see [the re
 <!-- PUT THE NEW CHANGELOG ENTRY RIGHT BELOW THIS -->
 <!-------------------------------------------------->
 
-## PLACEHOLDER FOR PORTAL WIDE METADATA
-
-* Metadata for all samples from all projects on the Portal can now be downloaded in a single tab-separated values file.
-* For more information on what to expect in the metadata file, see the {ref}`metadata section of the Downloadable files page <download_files:metadata>`.
```

Comment:
I also need to add this back to `development`.

Reply:
Seems reasonable to do this in a separate PR, rather than dealing with a messy branch just for this merge.